### PR TITLE
Actually fix Section links

### PR DESCRIPTION
**Pull request by magni-**
This time, with manual testing 🤦‍♂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
 ## Sections
-* [Prerequisites](##Prerequisites)
-* [Base Usage](##Base%20Usage)
-* [Optional Parameters](##Optional%20Parameters)
-* [Commands](##Commands)
-* [Executors](##Executors)
-* [Examples](##Examples)
-* [Orb Release Process](##Orb%20Release%20Process)
+* [Prerequisites](#prerequisites)
+* [Base Usage](#base-usage)
```
Comment (prayogoa) [inline README.md]: this still doesn't work

Comment (magni-) [reply README.md]: Good catch I forgot that one 🙈

Comment (magni-) [reply README.md]: Fixed

**Pull request merged**